src/task4feedback/simulator/rl/networks/a2c_fcn.py

```diff
@@ -0,0 +1,45 @@
+import torch
+import torch.nn
+import torch.nn.functional as F
+from torch.nn import Linear
+from torch_geometric.nn import GCNConv
+
+
+class A2CNetworkNoGCN(torch.nn.Module):
+
+    def __init__(self, in_dim: int, out_dim: int):
```

Comment:
Is it okay that the Actor and Critic are separate networks? 
"the literature" seems divided on this, with arguments both ways haha 
I don't have the experience to recommend either just curious to note it.

Reply:
I think this should be renamed to something new.. not A2C. A2C is not about a specific network, but more RL architecture. For example, we consider state-action probs and state value, but this uses return which is (state-action probs - state value). A2C needs to use 2 networks, actor and critic. But it doesn't matter what network they internally use. Sorry I just didn't know how to name it :-\